### PR TITLE
feature: tag sorting

### PR DESCRIPTION
**Pull request by dmitrakovich**

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -1,5 +1,5 @@
 parameters:
-  phpVersion: 70400
+  phpVersion: 80100
```
Comment (romanzipp) [inline phpstan.neon.dist:2]: The minimum PHP version in the `composer.json` is still 8.0 so I would like to keep it consistent here.

```suggestion
  phpVersion: 80000
```

File: src/Services/Tags/MixinTag.php [hunk 2]
```diff
@@ -6,6 +6,14 @@
...
 class MixinTag extends AbstractDescriptionTag
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function getTagName(): string
+    {
+        return 'mixin';
+    }
+
```
Comment (romanzipp) [inline src/Services/Tags/MixinTag.php:16]: I'm not sure why the mixin class would need a tag name

**Review by romanzipp — CHANGES_REQUESTED** [review]

File: src/Services/Tags/MixinTag.php (hunk 2 above)
Comment (dmitrakovich) [reply src/Services/Tags/MixinTag.php:16]: for short name in sorting list, without this, you need use 'romanzipp\modeldoc\services\tags\mixin'

Comment (romanzipp) [reply src/Services/Tags/MixinTag.php:16]: Alright, got it!

**Review by romanzipp — APPROVED** [review]

**Pull request merged**